data/projects/maci-platform.ts

```diff
@@ -1,13 +1,13 @@
 import { ProjectInterface } from "@/lib/types"
 
-export const maciRPGF: ProjectInterface = {
-  id: "maci-rpgf",
+export const maciPLATFORM: ProjectInterface = {
```

Comment:
try and change this to `maciPlatform` so that you won't have to change in the other file